docs/src/code-examples-content.js

```diff
@@ -1,610 +0,0 @@
-
```

Comment:
deleting this seems unintentional ?

Reply:
This file is rebuilt in production, so doesn't need to be checked in (although checking it in won't cause any harm)